src/box/lua/config/instance_config.lua

```diff
@@ -1629,6 +1629,7 @@ return schema.new('instance_config', schema.record({
         roles = schema.set({
             'router',
             'storage',
+            'rebalancer',
```

Comment:
Say, no rebalancer role assigned: does it mean that there is no rebalancer or that the rebalancer is chosen automatically? If the former, how to ask for the automation? If the latter, how to disable the rebalancer at all?

Reply:
For now, this means that the rebalancer will be selected from among all replicasets masters. Additionally, there is currently no option to disable the rebalancer via config.

Reply:
It is OK for now, but I think that we should provide ability to completely disable the rebalancer for administration tasks later (system, tarantool or application upgrade; disaster recovery).